tests/functional/aws-node-sdk/test/bucket/testBucketVersioning.js

```diff
@@ -146,11 +145,40 @@ describe('aws-node-sdk test bucket versioning', function testSuite() {
 
     it('should retrieve the valid versioning configuration', done => {
         const params = { Bucket: bucket };
-        // s3.getBucketVersioning(params, done);
         s3.getBucketVersioning(params, (error, data) => {
             assert.strictEqual(error, null);
             assert.deepStrictEqual(data, { Status: 'Enabled' });
             done();
         });
     });
 });
+
+describe.only('aws-node-sdk test bucket versioning with object lock', () => {
```

Comment:
@ilkescality You may have forgotten `.only` here

Reply:
ah!! just removed it, thanks!